crates/holochain_p2p/tests/tests/op_store.rs

```diff
@@ -139,6 +138,10 @@ impl HcP2pHandler for StubHost {
         unimplemented!()
     }
 
+    fn handle_publish_countersign(&self, _dna_hash: DnaHash, _op: ChainOp) -> BoxFut<'_, HolochainP2pResult<()>> {
```

Comment:
This is now a separate operation to publish. That cleans up quite a bit of code.